lib/testing_api_onlyoffice_com/test_instance/main_page/document_builder_api/document_builder_api_common/document_builder.rb

```diff
@@ -7,18 +7,41 @@ module DocumentBuilder
     include PageObject
 
     DEFAULT_BUILDER_FILE_NAME = 'SampleText.docx'.freeze
+    DEFAULT_BUILDER_DOCX_FILE_NAME = 'Sample.docx'.freeze
+    DEFAULT_BUILDER_XLSX_FILE_NAME = 'Sample.xlsx'.freeze
 
     # actions
     link(:generate_document, xpath: '//*[@id="generateButton"]')
+    link(:create_document, xpath: '//*[@id="createDocx"]')
+    link(:create_spreadsheet, xpath: '//*[@id="createXlsx"]')
 
-    def builder_works?
+    def builder_download
```

Comment:
bettern name will be `generate_document_from_script`
Also add documentation of return value

```
# @return [String] path to generated file
def generate_document_from_script
```